Time-Locked Accounts doc for change to convert times and dates into global slots (#355)
o1-labs/docs2: docs/zkapps/advanced-snarkyjs/time-locked-accounts.mdx

  />
</figure>

The red cross on the left marks the point in time where the `timing` field is set and `isTimed` switches from `false` to `true`. The orange line shows how the amount of unlocked tokens increases over time until it finally reaches its maximum value and stays flat. At this point, `isTimed` flips from `true` back to `false` because no tokens remain locked.

> As shown, the maximum amount of unlocked tokens is defined by the `initialMinimumBalance`. The property is called "initialMinimumBalance" because, even though the tokens show up in the balance, they can't be withdrawn. The account has a a non-zero _minimum balance_. Initially, that minimum balance is equal to the amount of tokens locked (for example, the initial minimum balance). Over time, the minimum balance decreases until it hits zero, which is the condition that makes `isTimed` false again.

Reviewer: ```suggestion
As shown, the maximum amount of unlocked tokens is defined by the `initialMinimumBalance`. The property is called "initialMinimumBalance" because, even though the tokens show up in the balance, they can't be withdrawn. The account has a a non-zero _minimum balance_. Initially, that minimum balance is equal to the amount of tokens locked (i.e., the initial minimum balance). Over time, the minimum balance decreases until it hits zero, which is the condition that makes `isTimed` false again.
```

Hey @barriebyron, **i.e.** (= id est) isn't meant in the sense of "for example", but in the sense of "that is" or "in other words". Not to be confused with "e.g.". Sometimes "i.e." can be replaced with "for example" but here it can't.

> Initially, that minimum balance is equal to the amount of tokens locked (i.e. the initial minimum balance).

This sentence was there to explain why we use the name `initialMinimumBalance` to refer to the total amount of tokens locked.
